src/components/Nav.js

```diff
@@ -1,20 +1,24 @@
-import { useState, useCallback } from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
 import styled, { css } from 'styled-components';
 
 const Nav = () => {
-  const [category, setCategory] = useState('');
-  const handleSelect = useCallback(category => setCategory(category), []);
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const moveTo = path => {
+    navigate(`${path}`);
+  };
 
   return (
     <NavWrapper>
-      <h2 className="logo">Yakurt</h2>
-      {NAV_DATA.map(navData => (
+      <Logo>Yakurt</Logo>
+      {NAV_DATA.map(({ id, text }) => (
         <Category
-          key={navData.id}
-          onClick={() => handleSelect(navData.id)}
-          active={category === navData.id}
+          key={id}
+          onClick={() => moveTo(id)}
+          active={`${location.pathname}` === id}
```

Comment:
```suggestion
          active={location.pathname === id}
```
string인 value를 다시 템플릿 리터럴 안에 넣을 필요는 없어보이네요!